pom.xml

```diff
@@ -222,7 +222,7 @@
                                         </goals>
                                     </pluginExecutionFilter>
                                     <action>
-                                        <ignore></ignore>
+                                        <ignore />
```

Comment:
Be careful when you do this.  Many parts of Maven quietly (or not so quietly) fail when there is not an explicit end tag.